packages/easy_logging/easy_logging.0.4/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Module to easily log messages"
+
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.0"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
```

Comment:
you can (not mandatory) also add `"-j" jobs` to the test command

Reply:
Thanks @kit-ty-kate. What's the end purpose of this parameter ? Is there any reason to limit the number of running jobs ?

Reply:
I think that if the number of jobs is not much higher than the CPU cores you do less context switching. I thought that withouth -j dune would use only a core, but reading the docs this seems to have changed